app/server/ruby/lib/sonicpi/util.rb

```diff
@@ -172,7 +174,15 @@ def raspberry_pi_4_8gb_64?
       os == :raspberry && @@raspberry_pi_4_8gb_64
     end
 
-    def unify_tilde_dir(path)
+      def raspberry_pi_400?
+      os == :raspberry && @@raspberry_pi_400
+    end
+
+     def raspberry_pi_400_64?
+      os == :raspberry && @@raspberry_pi_40_64
```

Comment:
@rbnpi should this be `@@raspberry_pi_400_64`?

Reply:
Oops yes, you are right. typo slip to which I am rather prone. I'll update.